ShareX.HelpersLib/Helpers/URLHelpers.cs

```diff
@@ -35,6 +35,7 @@
 using System.Text.RegularExpressions;
 using System.Threading.Tasks;
 using System.Web;
+using System.Windows.Forms;
```

Comment:
🤔

Reply:
Oh yeah I believe IDE autocompleted that when I was typing something, I'll remove it.